components/admin_console/billing/billing_subscriptions/index.tsx

```diff
@@ -118,6 +95,19 @@ const BillingSubscriptions: React.FC = () => {
     }
 
     useEffect(() => {
+        if (product !== null || isEmpty(products) || !subscription?.product_id) {
+            return;
+        }
+        const keys = Object.keys(products!);
+        let tempProduct: Product;
+        tempProduct = products![subscription?.product_id];
+        if (isEmpty(tempProduct)) {
+            tempProduct = products![keys[0]];
+        }
+        setProduct(tempProduct);
+    }, [products, subscription]);
+
+    useLayoutEffect(() => {
```

Comment:
Why `useLayoutEffect` and not good ol' `useEffect`?

Reply:
Asking based on what I read here https://kentcdodds.com/blog/useeffect-vs-uselayouteffect

Reply:
actually that was a leftover after fighting with the subscriptions information that ended up seeming as a chrome debugger problem, but yes, basically there is no difference between both hooks, however useLayoutEffects executes itself before useEffect, so I was using that situation but the end result ended up being the same. Let me set it back to useEffect since it makes no difference. Good catch @AGMETEOR !